server/chat.ts

```diff
@@ -1141,6 +1116,53 @@ export class CommandContext extends MessageContext {
 
 		return message;
 	}
+	canPM(targetUser: User) {
+		const user = this.user;
+
+		// for hotpatching, remove later.
+		const curSettings = Chat.updateSettings(user); // will just be user.settings
+		const tarSettings = Chat.updateSettings(targetUser); // will just be targetUser.settings
+
+		if (targetUser.locked && !user.can('lock')) {
+			this.errorReply(`The user "${targetUser.name}" is locked and cannot be PMed.`);
+			return null;
+		}
+		if (
```

Comment:
I seem to recall @Zarel not liking braces to be styled this way (and below - though I could very well be misremembering, if so, I apologize)?

Reply:
No, this is fine. What I don't like is if they don't match (i.e. there should be a newline after `(` if and only if there's a newline before the matching `)`)